sdk/src/utils.ts

```diff
@@ -194,26 +186,15 @@ export const deployPSP22 = async (
   decimals: bigint,
   network: Network
 ): Promise<PSP22> => {
-  return PSP22.getContract(
-    api,
-    network,
-    null,
-    DEFAULT_REF_TIME,
-    DEFAULT_PROOF_SIZE,
-    account,
-    supply,
-    name,
-    symbol,
-    decimals
-  )
+  return PSP22.getContract(api, network, account, supply, name, symbol, decimals)
 }
 
 export const deployWrappedAZERO = async (
   api: ApiPromise,
   account: IKeyringPair,
   network: Network
 ): Promise<WrappedAZERO> => {
-  return WrappedAZERO.getContract(api, account, null, DEFAULT_REF_TIME, DEFAULT_PROOF_SIZE, network)
+  return WrappedAZERO.getContract(api, network, account, process.env.WAZERO_ADDRESS)
```

Comment:
It's probably mistake to include both the deployer and contract address in the get contract call